apps/site/src/_pages/docs/1_developing-blocks.mdx

```diff
@@ -12,35 +12,30 @@ This guide helps get you set up and introduces some of the key features of the g
 
 In practice, most block developers will not need to know the lower-level details of the specifications, as the libraries we provide implement them.
 
-## Choosing your approach
-
-We provide three templates which allow you to define the entry point for your block in different ways:
+## Creating a block
 
-- `custom-element`: create a block defined as a custom element (also known as Web Components).
-- `html`: create a block defined as an HTML file, with JavaScript added via `<script>` tags.
-- `react`: create a block defined as a React component.
+1.  Move to a folder where you want to create your block.
+1.  Run `npx create-block-app@latest your-block-name`.
+1.  Switch to your new folder: `cd [your-block-name]`.
+1.  Run `yarn install && yarn dev` or `npm install && npm run dev`.
+1.  Open [http://localhost:63212](http://localhost:63212/) in your browser to see the starter template.
 
-To create a new block, run `npx create-block-app block-name --template template`, replacing `block-name` with the name to give your block, and `template` with one of the template names listed above.
 
 ### I want to use a different technology
 
-If you want to write blocks **using other technologies or frameworks**, you have several options:
+If you want to write blocks **using other technologies or frameworks**, you may pass a template parameter to the above command: `npx create-block-app@latest your-block-name --template <TEMPLATE>`. For the template, there are several options:
 
-1.  use a `custom-element` template and use different approaches when constructing the element-i.e. use a custom element as a wrapper for anything else you like.
-1.  use an `html` template and import and use your additional libraries inside the `<script>` tag.
-1.  use a language which can be transpiled to JavaScript. As an example, see [this blog post](https://hash.dev/blog/build-blocks-in-any-language) on writing a block using F#. This block uses React – you can use transpiled JavaScript in blocks defined as HTML files or custom elements too.
+1.  use the `react` template and use React to create your block (default).
+1.  use the `html` template and import and use your additional libraries inside the `<script>` tag.
+1.  use the `custom-element` template and use different approaches when constructing the element-i.e. use a custom element as a wrapper for anything else you like.
 
-### I don’t want to use TypeScript
+You may also use any other language which can be transpiled to JavaScript. As an example, see [this blog post](https://hash.dev/blog/build-blocks-in-any-language) on writing a block using F#. This block uses React – you can use transpiled JavaScript in blocks defined as HTML files or custom elements too.
 
-You can write your block in regular JavaScript using the methods described above - just rename your files from `*.tsx/*.ts` to `*.jsx/*.js`, remove the types, and get coding.
+The sections below assume, that you're using the `react` template.
```

Comment:
I think the only difference which isn't called out is the folder structure – can we delete this, but instead say where we talk about the folder structure something like:

> For HTML templates, the folder structure is slightly different. The development server is in a `dev/` folder and the block entry point is `app.html`

Reply:
Also, the file names are different (`.js` instead of `tsx`). There is no codegen in HTML blocks and you have to insert all the things manually.

Reply:
Can we call that out in the relevant place please? I guess where it talks about `block-metadata.json`

We mention the lack of TS generation already
> Once you have created the type representing the data your block needs, copy its URL, and update the `schema` property
in the `blockprotocol` object in `package.json`. In TypeScript block templates, you can then run `yarn schema` to
regenerate the types for your block.

Reply:
I tried to apply it in bbe1dd067d6082379d432d51c014ac09304abb8f